lib/http/client.rb

```diff
@@ -24,7 +24,12 @@ def initialize(default_options = {})
       @state = :clean
     end
 
-    # Make an HTTP request
+    # Make an HTTP request.
+    # @see Options
+    # @see https://ruby-doc.org/stdlib-2.5.1/libdoc/openssl/rdoc/OpenSSL/SSL/SSLContext.html OpenSSL::SSL::SSLContext for more information on :ssl options.
```

Comment:
Please, remove this line. `:ssl` is no better than any other options you can set with `Options`, and I see you have already added documentation there.

```suggestion
```

Reply:
Where in the doc do you say that the keys for the CA file are `{ :ssl => { :ca_file => '...' } }`?

Reply:
That's why I said that it should be in the options documentation not here.